presets/bat-master.json

```diff
@@ -0,0 +1,1514 @@
+    "address": "0x51EB318",
+    "value": "0x0C038C00"
+  }, {
+    "type": "word",
+    "value": "0x00000000"
+  }, {
+    "type": "word",
+    "address": "0x4FC717C",
+    "value": "0x0C038C00"
+  }, {
+    "type": "word",
+    "value": "0x00000000"
+  }, {
+    "type": "word",
+    "address": "0x507E548",
+    "value": "0x0C038C00"
+  }, {
+    "type": "word",
+    "value": "0x00000000"
+  }, {
+    "type": "word",
+    "address": "0x5135D08",
+    "value": "0x0C038C00"
+  }, {
+    "type": "word",
+    "value": "0x00000000"
+  }, {
+    "type": "word",
+    "address": "0x52C2B9C",
+    "value": "0x0C038C00"
+  }, {
+    "type": "word",
+    "value": "0x00000000"
+  }, {
+    "type": "word",
+    "address": "0x59389BC",
+    "value": "0x0C038C00"
+  }, {
+    "type": "word",
+    "value": "0x00000000"
+  }, {
+    "type": "word",
+    "address": "0x59F1BE8",
+    "value": "0x0C038C00"
+  }, {
+    "type": "word",
+    "value": "0x00000000"
+  }, {
+    "type": "word",
+    "address": "0x5780228",
+    "value": "0x0C038C00"
+  }, {
+    "type": "word",
+    "value": "0x00000000"
+  }, {
+    "type": "word",
+    "address": "0x4D38D18",
+    "value": "0x0C038C00"
+  }, {
+    "type": "word",
+    "value": "0x00000000"
+  }, {
+    "type": "word",
+    "address": "0x4DC65DC",
+    "value": "0x0C038C00"
+  }, {
+    "type": "word",
+    "value": "0x00000000"
+  }, {
+    "type": "word",
+    "address": "0x4E708C4",
+    "value": "0x0C038C00"
+  }, {
+    "type": "word",
+    "value": "0x00000000"
+  }, {
+    "type": "word",
+    "address": "0x4F0D0F8",
+    "value": "0x0C038C00"
+  }, {
+    "type": "word",
+    "value": "0x00000000"
+  }, {
+    "type": "word",
+    "address": "0x580A20C",
+    "value": "0x0C038C00"
+  }, {
+    "type": "word",
+    "value": "0x00000000"
+  }, {
+    "type": "word",
+    "address": "0x5139A68",
+    "value": "0x0C038C00"
+  }, {
+    "type": "word",
+    "value": "0x00000000"
+  }, {
+    "type": "word",
+    "address": "0x4AE430C",
+    "value": "0x0C038C00"
+  }, {
+    "type": "word",
+    "value": "0x00000000"
+  }, {
+    "type": "word",
+    "address": "0x4964E94",
+    "value": "0x0C038C00"
+  }, {
+    "type": "word",
+    "value": "0x00000000"
+  }, {
+    "type": "word",
+    "address": "0x5503944",
+    "value": "0x0C038C00"
+  }, {
+    "type": "word",
+    "value": "0x00000000"
+  }, {
+    "type": "word",
+    "address": "0x46DA788",
+    "value": "0x0C038C00"
+  }, {
+    "type": "word",
+    "value": "0x00000000"
+  }, {
+    "type": "word",
+    "address": "0x4FE3B80",
+    "value": "0x0C038C00"
+  }, {
+    "type": "word",
+    "value": "0x00000000"
+  }, {
+    "type": "word",
+    "address": "0x594B634",
+    "value": "0x0C038C00"
+  }, {
+    "type": "word",
+    "value": "0x00000000"
+  }, {
+    "type": "word",
+    "address": "0x5A7CE10",
+    "value": "0x0C038C00"
+  }, {
+    "type": "word",
+    "value": "0x00000000",
+    "comment": "End of marking doors for bat opening them"
+	}]
```

Comment:
*smh* fix indentation

Reply:
All tabs have been murdered and replaced.